addons/website/static/src/snippets/s_hr/000.scss

```diff
@@ -13,3 +13,17 @@
         background-color: transparent;
```

Comment:
> When dropping the Separator as an inner content, it still has the Background and Visibility options, is it a wanted behavior ? If not, shouldn't some code be added in a computeWidgetVisibility or something like that in order to hide them ? 🙂

It was decided with QSM to leave it like that in this PR. A [task](https://www.odoo.com/web#cids=1&menu_id=4720&action=333&active_id=974&model=project.task&view_type=form&id=3815935) has been created to fix that ;)

> Shouldn't Separator become a section too, in order to be consistent ?

It was decided with QSM to keep a div, the main reason being to avoid adapting all the XML containing separators and adding an upgrade script.

> It would also have the "Height" option by default, which I think could be useful with this snippet 

I don't see the point of defining the height of this snippet and I don't find it difficult to restore the padding.

> For the already dropped snippets from previous versions, we only need to change the data-vxml attribute in the snippet definition, in order to mark them as deprecated. Note that the user will then normally have the possibility to replace them by the new version 😉

I don't think you want to bother users with this just to put section instead of div.

> Shouldn't the block be moved in another section in the right panel then, since it is not only an inner content anymore ?

That's already been discussed too ;) we'll leave the snippet there, anyway it seems that an upcoming task will change the whole order of the snippet menu.

Reply:
@qsm-odoo Could you confirm the above points ? Thanks

Reply:
> It was decided with QSM to leave it like that in this PR. A [task](https://www.odoo.com/web#cids=1&menu_id=4720&action=333&active_id=974&model=project.task&view_type=form&id=3815935) has been created to fix that ;)

I don't think you made the same thing. I don't have the background-color option for countdown if I drop them as inner snippet. The annoying part is not being able to remove that color if I later move it as an inner snippet. This can indeed be fixed later. But your branch here seems to add the possibility to change background color of separator directly dropped as inner snippet... if there is some problem implementing the same as countdown, then maybe it is better to handle your fix task first :shrug: 

> It was decided with QSM to keep a div, the main reason being to avoid adapting all the XML containing separators and adding an upgrade script.

Agreed, but that's not the main reason. It's a bonus but IMO it's just that this should not be `<section>`.

> The section element represents a generic section of a document or application. A section, in this context, is a thematic grouping of content, typically with a heading.
> [...]
> The section element is not a generic container element. When an element is needed only for styling purposes or as a convenience for scripting, authors are encouraged to use the div element instead. A general rule is that the section element is appropriate only if the element's contents would be listed explicitly in the document's outline.

Using a section is already dubious for many of what we do but it is debatable: you could have a title associated to your form / countdown, using a section would then be nice. Without title, it's probably not really right... but it probably does not hurt. For a separator, I don't think it would ever make sense to be a section.

In the future, we'll probably have something to:
- Not have specific behavior for `<section>` but for `[data-snippet]` or specific classes
- Properly use `<div>` / `<section>` / others automatically

... but definitely not a priority. For now, let's use `section` most of the time and `div` when it does not make sense to use section for sure (like here).

> I don't see the point of defining the height of this snippet

Agreed, paddings are enough.

> I don't think you want to bother users with this just to put section instead of div.

I probably would but no debate here since we keep the `div`?

>  it seems that an upcoming task will change the whole order of the snippet menu.

Indeed. More than that actually, surprise :wink: